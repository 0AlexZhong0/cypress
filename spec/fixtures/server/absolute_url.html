<html>
  <head>
    <meta charset="utf-8">
    <title>absolute url test</title>
  </head>
  <body>
    <nav>
      <ul>
        <li>
          <a href="/relative.html">absolute path relative</a>
        </li>
        <li>
          <a href="../relative.html">path relative</a>
        </li>
        <li>
          <a href="regular/relative.html">relative</a>
        </li>
      </ul>
    </nav>
    <div>
      <form action="https://github.com/submit">
        <a href="http://www.google.com">google</a>
      </form>
      <a href="http://getbootstrap.com/css">css</a>
    </div>
  </body>
</html>