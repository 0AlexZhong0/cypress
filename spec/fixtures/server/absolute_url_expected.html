<html>
  <head>
    <script type='text/javascript'>
      window.onerror = function(){
        parent.onerror.apply(parent, arguments);
      }
    </script>
    <script type='text/javascript' src='/eclectus/js/sinon.js'></script>
    <script type='text/javascript'>
      var Cypress = parent.Cypress;
      if (!Cypress){
        throw new Error('Cypress must exist in the parent window!');
      };
      Cypress.onBeforeLoad(window);
    </script>
    <meta charset="utf-8">
    <title>absolute url test</title>
  </head>
  <body>
    <nav>
      <ul>
        <li>
          <a href="/relative.html">absolute path relative</a>
        </li>
        <li>
          <a href="../relative.html">path relative</a>
        </li>
        <li>
          <a href="regular/relative.html">relative</a>
        </li>
      </ul>
    </nav>
    <div>
      <form action="/https://github.com/submit">
        <a href="/http://www.google.com">google</a>
      </form>
      <a href="/http://getbootstrap.com/css">css</a>
    </div>
  </body>
</html>